### DISCUSSION TITLE
How do I add a modded light with varying light levels?

### DISCUSSION THREAD

**Discussion by LucaDon-G** [gri573/rethinking-voxels#223] [Q&A] [answered]
I am trying to add a light from the mod "Ar's Nouveau" its item id is "ars_nouveau:light_block" I have tried using "ars_nouveau:light_block:level=14" and just  "ars_nouveau:light_block" but I am having no luck. I am trying to make sure that it emits light at all levels.

**Comment by DarianLStephens** [comment, marked as answer]
I don't know for sure if this limitation still applies, but it used to be that the shader couldn't emit light from blocks without physical geometry. Particles don't count. This is why the light blocks couldn't emit light, but I believe they fixed that recently?

**Reply by gri573** [reply]
light blocks were special-cased by iris to allow them to work. this does not apply to any modded block with no geometry, so those don't work
